packages/liveblocks-node-lexical/package.json

```diff
@@ -39,11 +39,11 @@
     "yjs": "^13.6.18"
   },
   "peerDependencies": {
-    "@lexical/headless": "0.16.0",
-    "@lexical/selection": "0.16.0",
-    "@lexical/yjs": "0.16.0",
-    "@lexical/markdown": "0.16.0",
-    "lexical": "0.16.0"
+    "@lexical/headless": "0.16.1",
+    "@lexical/selection": "0.16.1",
+    "@lexical/yjs": "0.16.1",
+    "@lexical/markdown": "0.16.1",
+    "lexical": "0.16.1"
```

Comment:
@jrowny Updated the version of these packages on `@liveblocks/node-lexical` to `0.16.1` too. Do you see any issues with it?

Reply:
Should these be using `^0.16.1` now, instead of `0.16.1` like we do in `@liveblocks/react-lexical`?

Reply:
@nvie I noticed they were pinned to the exact `0.16.0` version. I do not recall why we did that. @jrowny Do you know why that was done and if this change might create issues or if it is okay to set it to `^0.16.1`?

Reply:
should be good, for node specifically, it needs to be locked, no ~ because of the fact that Lexical stores globals and we can't have different versions between what we build and what the user has.